index.php

```diff
@@ -21,13 +21,18 @@
  * @copyright  2020 Catalyst IT
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
+require_once(__DIR__.'/../../vendor/autoload.php');
```

Comment:
lets pull this line out - it won't be needed in 3.10 or higher branches as zipstream is already in the autoload list somewhere... and it's not really the right way to include zipstream for older branches.